core/src/main/resources/error/error-classes.json

```diff
@@ -14,13 +18,25 @@
   "CANNOT_EVALUATE_EXPRESSION" : {
     "message" : [ "Cannot evaluate expression: %s" ]
   },
+  "CANNOT_FETCH_TABLES_OF_DATABASE" : {
```

Comment:
We can simplify this: `CANNOT_FETCH_TABLES_OF_DATABASE` -> `CANNOT_FETCH DATABASE_TABLES`